week-3/slideshow/slideshow.js

```diff
@@ -5,4 +5,35 @@ const images = [
 ];
 
 
-// Write your code here
\ No newline at end of file
+// Write your code here
+const autoBackwardBtn = document.getElementById("auto-backward-btn");
+const backwardBtn = document.getElementById("backward-btn");
+const stopBtn = document.getElementById("stop-btn");
+const forwardBtn = document.getElementById("forward-btn");
+const autoForwardBtn = document.getElementById("auto-forward-btn");
+const carouselImages = document.getElementById("carousel-img");
+
+let currentIndex = 1;
+let autoIntervalID;
+
+function backward() {
+  currentIndex = (currentIndex -1 + images.length) % images.length;
+  carouselImages.src = images[currentIndex];
+}
+
+function forward() {
+    currentIndex = (currentIndex + 1) % images.length;
+    carouselImages.src = images[currentIndex];
+}
+
```

Comment:
Your code follows consistent naming conventions which is easy to follow and understand.

Reply:
Thank you for your feedback